code/src/qik_2s12v10.cpp

```diff
@@ -114,20 +114,29 @@ namespace r2d2::moving_platform {
         usart_bus << qik_2s12v10_registers::autodetect_baudrate;
     }
 
-    void qik_2s12v10_c::brake(const int8_t brake_amount) {
+    void qik_2s12v10_c::brake(uint8_t brake_amount) {
```

Comment:
Why doesn't this function call the `brake_m0` and `brake_m1` functions?

Reply:
I agree that this can be done. However, these functions were already implemented by someone else (as this code was already on master) and this was not the scope of this PR.

Reply:
Changed the function in the last commits. Can you take another look?